xbuild/src/cargo/mod.rs

```diff
@@ -10,8 +10,14 @@ mod utils;
 
 pub use artifact::{Artifact, CrateType};
 
+use self::config::Config;
+use crate::{Arch, CompileTarget, Opt, Platform};
+
 pub struct Cargo {
     package: String,
+    // TODO: Since xbuild doesn't have the cargo-subcommand+cargo-apk split we might get away
+    // with reusing the same Manifest deserialized type here?
+    workspace_manifest: Option<PathBuf>,
```

Comment:
wokspace_manifest is unesed, will it be used later?

Reply:
Correct, I may want to `_` this for now.